cmd/nerdctl/run_security_linux.go

```diff
@@ -35,8 +35,6 @@ import (
 
 var privilegedOpts = []oci.SpecOpts{
 	oci.WithPrivileged,
-	oci.WithAllDevicesAllowed,
```

Comment:
Curious: Does this affect network devices as well? How will network connectivity work inside the VM?

Reply:
Kata has its route to set up network between host and guest without /dev/xx device. If additional devices are needed, that can be specified by `--device` option.